### ISSUE TITLE
Get tests green on refactor branch

### ISSUE THREAD

**Issue by miketaylr** [webcompat/webcompat.com#2106]
Once this is done, no more PRs into refactor will be allowed without fixing broken tests.

**Comment by miketaylr** [comment]
```
× firefox on darwin 17.4.0 - Comments (auth) - Posting a comment
× firefox on darwin 17.4.0 - Comments (auth) - Add a screenshot to a comment
× firefox on darwin 17.4.0 - Comments (auth) - Pressing 'l' inside of comment textarea *doesn't* open the label editor box
× firefox on darwin 17.4.0 - Contributors - page loads
× firefox on darwin 17.4.0 - Contributors - clicking first section closes it
× firefox on darwin 17.4.0 - Contributors - clicking section toggles it
× firefox on darwin 17.4.0 - Contributors - toggling section toggles lightbulb
× firefox on darwin 17.4.0 - History navigation - Back button works from issues page
× firefox on darwin 17.4.0 - Image Uploads (non-auth) - remove image upload button
× firefox on darwin 17.4.0 - Image Uploads (non-auth) - double image select works
× firefox on darwin 17.4.0 - Index - front page loads
× firefox on darwin 17.4.0 - Index - reporter addon link is shown
× firefox on darwin 17.4.0 - Index - form toggles open then closed
× firefox on darwin 17.4.0 - Index - browse issues (needstriage)
× firefox on darwin 17.4.0 - Issue-list - FilterView renders
× firefox on darwin 17.4.0 - Issue-list - IssueListView renders
× firefox on darwin 17.4.0 - Issue-list - Dropdowns reflect state from URL
× firefox on darwin 17.4.0 - Issue-list - Going back in history updates issue list and URL state
× firefox on darwin 17.4.0 - Issue-list - Clicking on a stage filter adds the correct param to the URL
× firefox on darwin 17.4.0 - Issue-list - Toggling a stage filter doesn't leave the param in the URL
× firefox on darwin 17.4.0 - Issue-list - Toggling between stage filters results in last param in URL
× firefox on darwin 17.4.0 - Issues (auth) - Pressing 'l' opens the label editor box
× firefox on darwin 17.4.0 - Issues - Issue page loads
× firefox on darwin 17.4.0 - Labels (auth) - Label editor opens then closes (key events)
× firefox on darwin 17.4.0 - Milestones (auth) - Milestone editor opens then closes (key events)
× firefox on darwin 17.4.0 - Milestones (non-auth) - Page loads without milestone set
× firefox on darwin 17.4.0 - New Issue Page - new issue page loads
× firefox on darwin 17.4.0 - Reporting (auth) - Report button shows name
× firefox on darwin 17.4.0 - Reporting (non-auth) - Submit buttons are disabled
× firefox on darwin 17.4.0 - Reporting (non-auth) - Report button shows via GitHub
× firefox on darwin 17.4.0 - Reporting (non-auth) - URL validation
× firefox on darwin 17.4.0 - Reporting (non-auth) - Description validation
× firefox on darwin 17.4.0 - Reporting (non-auth) - (optional) browser + os validation
× firefox on darwin 17.4.0 - Reporting (non-auth) - Image extension validation
× firefox on darwin 17.4.0 - Reporting (non-auth) - Submits are enabled
× firefox on darwin 17.4.0 - Reporting (non-auth) - problem_type param selects problem type
× firefox on darwin 17.4.0 - Search (auth) - Search/filter interaction
× firefox on darwin 17.4.0 - Search (auth) - Results are loaded from the query params
× firefox on darwin 17.4.0 - Search (auth) - Search works by icon click
× firefox on darwin 17.4.0 - Search (auth) - Search works by Return key
× firefox on darwin 17.4.0 - Search (auth) - Search from the homepage
× firefox on darwin 17.4.0 - Search (auth) - Search with a dash works
× firefox on darwin 17.4.0 - Search (auth) - Search input is loaded from q param (with dashes)
× firefox on darwin 17.4.0 - Search (non-auth) - Clicking on label search updates the search input
× firefox on darwin 17.4.0 - Search (non-auth) - Empty search input removes q param
× firefox on darwin 17.4.0 - User Activity (auth) - We're at the right place
× firefox on darwin 17.4.0 - User Activity (auth) - IssueListView renders
× firefox on darwin 17.4.0 - User Activity (auth) - Trying to view someone else's activity fails (logged in)
× firefox on darwin 17.4.0 - User Activity (non-auth) - Trying to view someone else's activity fails 
```

**Comment by miketaylr** [comment]
https://github.com/webcompat/webcompat.com/commit/82f1e7fb8cb40678ed77c4727054f035e241957e

**Comment by miketaylr** [comment]
OK, this is done. But possibly the tests are flaky now. Please file follow up bugs for those tests, if anyone runs into them locally.
